lib/utils/__tests__/findAtRuleContext.test.js

```diff
@@ -5,31 +5,26 @@ const postcss = require('postcss');
 
 it('findAtRuleContext', () => {
 	const css = `
-    a {}
-    @media print {
-      b {}
-    }
-    @media (min-width: 900px) {
-      c {}
-    }
-    d {}
-  `;
+	a {}
```

Comment:
Could increase all template literal content indentation by one level? So it look nicer :)

Reply:
Agreed. I'll fix the template literals in other files in another PR. 💪